### DISCUSSION TITLE
Help understand this huge random dip

### DISCUSSION THREAD

**Discussion by SodaSyrup** [ghostfolio/ghostfolio#3712] [Q&A]
Hello, i have a huge one day dip in my overview graph, none of my accounts ever were in the negative.

![image](https://github.com/user-attachments/assets/b89f8066-7f46-49ad-a0af-521503aa75d7)

**Comment by SodaSyrup** [comment]
on the accounts tab graph for the only account that had activity there, it also weirdly shows 0, but the day before and after are normal again?


![image](https://github.com/user-attachments/assets/5a3311a1-fd4d-4380-9ea0-da9d1f6e803f)


![image](https://github.com/user-attachments/assets/abdb2ace-eb41-41ab-842b-fb2c44269495)

![image](https://github.com/user-attachments/assets/753df579-51d2-4546-bf9d-0c8b07648999)

**Comment by mchal** [comment]
Have you checked `Admin Control > Market Data` for the investments that you have for that specific day? 
Maybe it needs to be overwritten with the correct value? 

Example what you are looking for - on this popup you will be able to click specific square for specific date to check what was the value.
<img width="811" alt="image" src="https://github.com/user-attachments/assets/0f3add7f-0e17-4f42-9421-ef1580b2eb61">

**Reply by SodaSyrup** [reply]
super late reply, but nope all good on that day and around it

**Reply by Dilosen** [reply]
This can happen if the stock is put on hold or if trading is suspended. But also, if you do big deposits or withdrawls.  Be sure to check the activities prior to the date for any anomalies.
